README.MD

```diff
@@ -0,0 +1,71 @@
+# Securedrop
+
+Securedrop is a command line application that facilitates the secure transfer of files between users in a peer-to-peer network.
+
+
+## Installation
+
+### Clone and Build
+```bash
+git clone https://github.com/aokellermann/securedrop
+cd ./securedrop
+python ./scripts/setup.py build
+```
+
+### Install dependencies 
+```bash
+python ./scripts/setup.py install
+```
+## Usage
+
+### Running with no executable arguments
+
+#### Running the server
+```bash
+./bin/securedrop_server
+```
+#### Running the client
+```bash
+./bin/securedrop
+```
+### Running with executable arguments
+#### Running the server
+```bash
+./bin/securedrop_server --port 6789 --file example_file.json --verbose
+```
+#### Running the client
+```bash
+./bin/securedrop --port 6789 --address 127.0.0.1 -f example_file_2.json --verbose 
+```
+
+## Code
```

Comment:
This is the type of stuff that should go in `.github/CONTRIBUTING.md`, if you want to include it. Regular users don't need to see it. See [here](https://github.com/maxitg/SetReplace/blob/master/.github/CONTRIBUTING.md) for what a good contributing file looks like.

Reply:
I did this and then you said don't do this.